node/packages/client/src/ethereum/parser/foxy.ts

```diff
@@ -0,0 +1,69 @@
+import { Tx as BlockbookTx } from '@shapeshiftoss/blockbook'
+import { ethers } from 'ethers'
+import { TxParser } from '../../types'
+import { TxSpecific, SubParser } from '../types'
+import { getSigHash, txInteractsWithContract } from './utils'
+import { FOXY_STAKING_CONTRACT } from './constants'
+import FOXY_STAKING_ABI from './abi/foxyStaking'
+
+export class Parser implements SubParser {
+  abiInterface: ethers.utils.Interface
+
+  readonly stakeSigHash: string
+  readonly unstakeSigHash: string
+  readonly instantUnstakeSigHash: string
+  readonly claimWithdrawSigHash: string
+
+  constructor() {
+    this.abiInterface = new ethers.utils.Interface(FOXY_STAKING_ABI)
+    // Hard coded staking sigHash because there is two different stake methods
+    // in the foxy staking contract. This gives us the ability to parser the correct one.
+    ;(this.stakeSigHash = '0x7acb7757'),
+      (this.unstakeSigHash = this.abiInterface.getSighash('unstake')),
+      (this.instantUnstakeSigHash = this.abiInterface.getSighash('instantUnstake')),
+      (this.claimWithdrawSigHash = this.abiInterface.getSighash('claimWithdraw'))
```

Comment:
Is there an advantage to using the comma operator in this way, over just doing the below?

```ts
this.stakeSigHash = '0x7acb7757'
this.unstakeSigHash = this.abiInterface.getSighash('unstake')
this.instantUnstakeSigHash = this.abiInterface.getSighash('instantUnstake')
this.claimWithdrawSigHash = this.abiInterface.getSighash('claimWithdraw')
```

Reply:
Nope, not at all. I believe this was a linter change. Will fix.